src/extension.ts

```diff
@@ -58,20 +58,19 @@ export function activate(context: vscode.ExtensionContext) {
 
   const clientOptions: vscodelc.LanguageClientOptions = {
     // Register the server for c-family and cuda files.
-    documentSelector: [
-      { scheme: 'file', language: 'c' },
-      { scheme: 'file', language: 'cpp' },
+    documentSelector : [
```

Comment:
so this formatting with space before the : is just wrong - it's not what clang-format is trying to do, it's just formatting [ ... ] as type-annotations because we've triggered a bug. Let me have a look at whether there's a good workaround...

Reply:
Actually this is a style option - it's just a terrible one.
See #11 - WDYT?